packages/framework-provider-azure/src/library/read-model-adapter.ts

```diff
@@ -18,7 +25,72 @@ export async function fetchReadModel(
     `[ReadModelAdapter#fetchReadModel] Loaded read model ${readModelName} with ID ${readModelID} with result:`,
     resource
   )
-  return [resource as ReadModelInterface]
+
+  if (!resource) return [resource as ReadModelInterface]
+
+  return [
+    {
+      ...resource,
+      boosterMetadata: {
+        ...resource?.boosterMetadata,
+        optimisticConcurrencyValue: resource._etag,
+      },
+    } as ReadModelInterface,
+  ]
+}
+
+async function insertReadModel(
+  logger: Logger,
+  readModel: ReadModelInterface,
+  db: CosmosClient,
+  config: BoosterConfig,
+  readModelName: string
+): Promise<void> {
+  const itemModel = {
+    ...readModel,
+    id: readModel.id.toString(),
+  } as ItemDefinition
+  try {
+    await db
+      .database(config.resourceNames.applicationStack)
+      .container(config.resourceNames.forReadModel(readModelName))
+      .items.create(itemModel)
+    logger.debug('[ReadModelAdapter#insertReadModel] Read model inserted')
+  } catch (err) {
+    // In case of conflict (The ID provided for a resource on a PUT or POST operation has been taken by an existing resource) we should retry it
+    if (err.code == 409) {
```

Comment:
Maybe setting a named constant for this code? There might be a `constants.ts` file somewhere..

```suggestion
    if (err.code == AZURE_CONFLICT_ERROR_CODE) {
```

Reply:
Agree